scriptmodules/admin/setup.sh

```diff
@@ -453,10 +453,10 @@ function config_gui_setup() {
 
 function update_packages_setup() {
     clear
-    local idx
-    for idx in ${__mod_idx[@]}; do
-        if rp_isInstalled "$idx" && [[ "${__mod_section[$idx]}" != "depends" ]]; then
-            rp_installModule "$idx" "_update_" || return 1
+    local id
+    for id in ${__mod_id[@]}; do
+        if rp_isInstalled "$id" && [[ "${__mod_section[$id]}" != "depends" ]]; then
+            rp_installModule "$id" "_update_" || return 1
```

Comment:
Maybe this should be handled similar to https://github.com/RetroPie/RetroPie-Setup/commit/0e63f091bd20569ab68f9f6bb259ababb245b176 - don't fail on a single package installation.

Reply:
I agree. I'll adjust that in a separate commit from the reworking so it's clear. Thanks.